<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>359</width>
      <height>102</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLineEdit" name="data_1">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="data_5"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="data_8"/>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="data_3"/>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="data_4"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="data_2"/>
     </item>
     <item row="4" column="3">
      <widget class="QLineEdit" name="data_9"/>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="data_6"/>
     </item>
     <item row="4" column="0">
      <widget class="QLineEdit" name="data_7"/>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="out_name"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>     添加合成</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="add_1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>添加有序合成</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>220</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>添加无序合成</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string> 删除合成</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="remove_data"/>
     </item>
     <item>
      <widget class="QPushButton" name="remove_rec">
       <property name="text">
        <string>点击删除</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
